src/lib/import-export/export/exporters/jetpack-exporter.ts

```diff
@@ -0,0 +1,103 @@
+import fs from 'fs';
+import fsPromises from 'fs/promises';
+import os from 'os';
+import path from 'path';
+import archiver from 'archiver';
+import { ExportOptions, BackupContents, Exporter } from '../types';
+
+export class JetpackExporter implements Exporter {
```

Comment:
The format technically won't be Jetpack format - it will be compatible with Jetpack but will support more features, e.g., `.zip` as an extension or `studio.json` metafile. Could we rename it to `DefaultExporter` then?